CanDIG/CanDIGv2
# DIG-1071: Revert "cleanup of conda related scripts that are no longer needed"

This is a simple `git revert` for commit dc209a53b8b1e9289b137ab6cc03002e7a3522d7 , and a simple documentation update.

**To test**:
- The build process needs to still work (instructions on [this version of the docs](https://github.com/CanDIG/CanDIGv2/blob/9a5be2859a8209ccf24f3422a17b67f1d26b4ed8/docs/install-docker.md), but with the latest versions of each submodule. I've tested building it myself, but should be confirmed on everyone's system.

- [x] MacOS X
- [x] MacOS X (M1)
- [x] Windows (WSL)
